### DISCUSSION TITLE
Set multiple timers with a single JSON string?

### DISCUSSION THREAD

**Discussion by max-allan** [arendst/Tasmota#20312] [Q&A]
Hi, if I use `timers` I get a neat list of all the timers in one JSON string.
If I try to put something like that JSON after the `timers` command, nothing gets set.
If I take the JSON value of `"Timer1"` and put it after the `Timer1` command, it does get set.

It'd be really handy if there was a way to take the JSON you get from `timers` and not have to do a load of editing to put it back in to configure the timers. (Imagine you want to change all the timers slightly. Not only do you need to change the timer, but you need to remove the colon and quotes around `"Timer?":`, and the final comma. Yes, I could probably script something with jq, but it feels like Tasmota could do this all for me.)

Is there a way to do that? The docs for Timers don't mention it. But maybe there is a different way to apply JSON configs?

**Comment by sfromis** [comment]
Not what you want, but the closest is that can issue multiple commands using `Backlog`, but that still requires splitting the long JSON into each of the timers as one command.

While there exists a command named `json`, which can take a JSON argument and generate a Backlog sequence for multiple commands, this only works for single command/value pairs, not for processing nested JSON.
`json {"SerialLog":2,"WebLog":3,"MqttLog":0,"SysLog":0}` gets transformed to each command.
